src/main/java/capstone/mfslbackend/model/Game.java

```diff
@@ -0,0 +1,61 @@
+package capstone.mfslbackend.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@ToString
+@Data
```

Comment:
Agree with Seb

Reply:
they're not there. maybe you're looking at the wrong version of the file?